src/frontend/src/lib/components/tokens/TokensSkeletons.svelte

```diff
@@ -3,9 +3,11 @@
 	import { fade } from 'svelte/transition';
 	import SkeletonCards from '$lib/components/ui/SkeletonCards.svelte';
 	import { TOKENS_SKELETONS_INITIALIZED } from '$lib/constants/test-ids.constants';
+
+	export let loading = false;
 </script>
 
-{#if $erc20UserTokensNotInitialized}
+{#if $erc20UserTokensNotInitialized || loading}
```

Comment:
If we introduce a property named `loading` then I would move `$erc20UserTokensNotInitialized` outside of the components and used it in the composition of the `loading` assignment.

Alternativey, I would not add a property loading but, a property "tokens" and add the logic "isNullish" here.

The first solution being probably cleaner but, I suggest to not mix logic between inner and outter sources.

Reply:
Yes, agreed! But maybe better to put it in another PR, to be cleaner?

https://github.com/dfinity/oisy-wallet/pull/2136